RoomsQuickstart/Program.cs

```diff
@@ -53,18 +60,10 @@ static async Task CreateRoom()
                 DateTimeOffset validFrom = DateTimeOffset.UtcNow;
                 DateTimeOffset validUntil = DateTimeOffset.UtcNow.AddDays(1);
 
-                if (roomsClient is not null)
-                {
-                    RoomModel createRoomResult = await roomsClient.CreateRoomAsync(validFrom, validUntil, RoomJoinPolicy.InviteOnly, roomParticipants, cancellationToken);
-                    rooms.Add(createRoomResult.Id);
-
-                    RoomModel roomInfo = await roomsClient.GetRoomAsync(createRoomResult.Id);
-                    PrintRoom(roomInfo);
-                }
-                else
-                {
-                    throw new Exception("roomsClient is null");
-                }
+                RoomModel createRoomResult = await roomsClient.CreateRoomAsync(validFrom, validUntil, RoomJoinPolicy.InviteOnly, roomParticipants, cancellationToken);
+                rooms.Add(createRoomResult.Id);
+                RoomModel roomInfo = await roomsClient.GetRoomAsync(createRoomResult.Id);
```

Comment:
We can use the createdRoom to print the room info, do we need to make getRoom call?

Reply:
No, we don't need GetRoom call, removed it